sovtokenfees/sovtokenfees/main.py

```diff
@@ -82,23 +79,8 @@ def register_req_handlers(node, fees_tracker):
 
 
 def register_batch_handlers(node, fees_tracker):
-    handlers = node.write_manager.batch_handlers[DOMAIN_LEDGER_ID]
-    node.write_manager.remove_batch_handler(DOMAIN_LEDGER_ID)
-
-    # Temp checks, remove after integration
-    if len(handlers) != 4 or not isinstance(handlers[-1], TsStoreBatchHandler) \
-            or not (handlers[-2], AuditBatchHandler):
-        raise LogicError
-
-    handlers.insert(handlers.index(handlers[-2]),
-                    DomainFeeBatchHandler(node.db_manager, fees_tracker))
-
-    for h in handlers:
-        if isinstance(h, (AuditBatchHandler, TsStoreBatchHandler)):
-            node.write_manager.register_batch_handler(h, ledger_id=DOMAIN_LEDGER_ID)
-        node.write_manager.register_batch_handler(h)
-    # TODO: Additional functionality to request_manager ^^^
-
+    domain_fee_batch_handler = DomainFeeBatchHandler(node.db_manager, fees_tracker)
```

Comment:
It looks like we need to put it before Audit Handler since Audit ledger tracks ledger root hashes and sizes for all ledger (two ledgers are changed if we order a domain txn with fees)